### ISSUE TITLE
npm ERR! note-wrangler@0.0.0 start: `node app.js`

### ISSUE THREAD

**Issue by Gregory-Phillips** [codeschool/NoteWrangler#3]
Thanks for making this public and I think this will be a great benefit to this Course. I'm currently at the start of Level 3 and really wanted to see this code working so everything is not so abstract.

I cloned and ran npm install... then I ran npm start following the instructions on the README. Here is the error: 

``````
module.js:340
    throw err;
          ^
Error: Cannot find module 'passport-local'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/***/_development/codeSchool/AngularJS/NoteWrangler/app/server/modules/routes/session.js:2:21)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)

npm ERR! note-wrangler@0.0.0 start: `node app.js`
npm ERR! Exit status 8
npm ERR!
npm ERR! Failed at the note-wrangler@0.0.0 start script.
npm ERR! This is most likely a problem with the note-wrangler package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node app.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls note-wrangler
npm ERR! There is likely additional logging output above.
npm ERR! System Darwin 14.1.0
npm ERR! command "node" "/usr/local/bin/npm" "start"
npm ERR! cwd /***/_development/codeSchool/AngularJS/NoteWrangler
npm ERR! node -v v0.10.31
npm ERR! npm -v 1.4.26
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /***/_development/codeSchool/AngularJS/NoteWrangler/npm-debug.log
npm ERR! not ok code 0 ```

Please let me know if I need to provide any additional information like the debug.log as mentioned in this error.

Thanks again,
G
``````

**Comment by dvdcastro** [comment]
I fixed this installing the passport-local module.

```
npm install passport-local
```

**Comment by fercreek** [comment]
dvdcastro I did that, but doesn't works.

**Comment by rgracia** [comment]
I am having the same problem. This is an epic fail. Will have to just review the STB version.

**Comment by jonfriskics** [comment]
Hi all, not sure where things broke down, but I just bumped the required versions of bcrypt and sqlite3 and things are installing fine for me now locally with `npm install`.  Try pulling down the latest version and giving it a shot - and thanks for your patience!

**Comment by gabrielfreiberg** [comment]
Hey all, having the same issue on OSX.  Tried installing passport-local per @dvdcastro's note above to no avail.

Update: something else was running on port 80 for me.  I killed that process and started again, works now.

**Comment by renz45** [comment]
This looks to be fixed, I updated everything for node 5.5. 👍
